src/menu.js

```diff
@@ -0,0 +1,39 @@
+const menu = () => {
+  const bannerel = document.createElement('div');
+  const jumbotronel = document.createElement('div');
+  const jumbotrontitleel = document.createElement('h1');
+  const jumbotronpel1 = document.createElement('p');
+  const jumbotronpel2 = document.createElement('p');
+  const jumbotronpel3 = document.createElement('p');
+  const jumbotronpel4 = document.createElement('p');
+  const jumbotronpel5 = document.createElement('p');
+  const jumbotronpel6 = document.createElement('p');
+  const jumbotronpel7 = document.createElement('p');
+  bannerel.classList.add('banner');
+  jumbotronel.classList.add('jumbotron', 'center');
+  jumbotrontitleel.classList.add('jumbotrontitleel');
+  jumbotronpel1.classList.add('lead');
+
+  jumbotrontitleel.textContent = 'Choose from Our Menu:';
+  jumbotronpel1.textContent = 'Continental';
+  jumbotronpel2.textContent = 'Japanese';
+  jumbotronpel3.textContent = 'Indian';
+  jumbotronpel4.textContent = 'Chainese';
+  jumbotronpel5.textContent = 'Italian';
+  jumbotronpel6.textContent = 'Dutch';
+  jumbotronpel7.textContent = 'Mexican';
+
+  bannerel.appendChild(jumbotronel);
+  jumbotronel.appendChild(jumbotrontitleel);
+  jumbotronel.appendChild(jumbotronpel1);
+  jumbotronel.appendChild(jumbotronpel2);
+  jumbotronel.appendChild(jumbotronpel3);
+  jumbotronel.appendChild(jumbotronpel4);
```

Comment:
- [ ] You also have lots of repeated codes here. And I also noticed this in almost all of your files as well, like the banner.js, chef.js, contact.js. I'll appreciate it if you kindly take your time to refactor these codes to look cleaner and maintain the DRY rule. This will definitely help you greatly in future projects and programming in general.

To get the highlighted code cleaned up, you can do the following. Since the paragraph (p) elements were created in several instances, you can remove all `document.createElement('p')`, remove all textContent of all paragraph elements, then remove all paragraph variables appended to `jumbotrone1` . Then create a loop expression that does all these without having to repeat yourself. Like this - 

```
 const list  = ['Continental', 'Japanese', 'Indian', 'Chainese', 'Italian', 'Dutch', 'Mexican'];
  list.forEach((li, ind) => {
    const paraElement = document.createElement('p');
    if (ind === 0) {
      paraElement.classList.add('lead');
    }
    paraElement.textContent = li;
    jumbotronel.appendChild(paraElement);
  })
```
Kindly use the method I provided or any other similar method you can create to refactor this code and other similar codes in other files.

Reply:
Thanks a lot for the suggestions, I have made the changes you asked here for banner.js, chef.js, contact.js.